tests/integration/test_monitor.py

```diff
@@ -0,0 +1,120 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+import gpflow
+from gpflow.utilities.monitor import (
+    ModelToTensorBoardTask,
+    ScalarToTensorBoardTask,
+    TasksCollection,
+    ImageToTensorBoardTask,
+)
+
+
+class Data:
+    num_data = 20
+    num_steps = 10
+    log_dir = tempfile.gettempdir()
```

Comment:
The test might be more robust with using pytest's `tmp_path` fixture, this will also handle eventually deleting them again. This could then be used to also test that a tensorboard was actually written out to disk?

Reply:
done